composer.json

```diff
@@ -20,6 +20,7 @@
   ],
   "require": {
     "silverstripe/framework": "^4@dev",
+    "intervention/image": "^2.3",
```

Comment:
This "fixes" the master build because it raises the PHPUnit dep from 4.x to 5.x, which in turn seems to fix a weird bug around invoking `@dataProvider` functions before calling test setup (which results in `No config manifests available`)